doc/api/esm.md

```diff
@@ -1096,20 +1099,152 @@ export async function resolve(specifier,
 With this loader, running:
 
 ```console
-NODE_OPTIONS='--experimental-loader ./custom-loader.mjs' node x.js
+NODE_OPTIONS='--experimental-loader ./custom-loader.mjs' node main.js
 ```
 
-would load the module `x.js` as an ES module with relative resolution support
+would load the module `main.js` as an ES module with relative resolution support
 (with `node_modules` loading skipped in this example).
 
+### Get source hook
```

Comment:
Perhaps just call this "source hook"?

Reply:
I think having two hooks `source` and `transformSource` is confusing. Assuming we have multiple source-related hooks I think they should each have distinct names.

Reply:
I was not suggesting changing the name of the hook, only the title of this section.